packages/x-date-pickers/src/locales/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './deDE';
 export * from './esES';
 export * from './faIR';
 export * from './fiFI';
+export * from './csCS';
```

Comment:
Correct me if I'm wrong, but I think that the [`BCP 47 language tag` is `cs-CZ`](https://www.techonthenet.com/js/language_tags.php).
This would mean that the file should have a `csCZ` name instead.
Naming it like this would probably also trigger a localization statistics script to pick up this locale, update the data json and include it [in the table](https://deploy-preview-7645--material-ui-x.netlify.app/x/react-date-pickers/localization/#supported-locales).